src/components/Input/Input.tsx

```diff
@@ -1,52 +1,88 @@
 import styled, { css } from 'styled-components'
 import { transparentize } from 'polished'
-import { ErrorMessage } from '@/components/'
+import { type ErrorFieldMessageProps } from '@/components/'
 import React from 'react'
+import dynamic from 'next/dynamic'
+
+const ErrorFieldMessage = dynamic<ErrorFieldMessageProps>(
+  () =>
+    import('@/components/ErrorFieldMessage/ErrorFieldMessage').then(
+      (module) => module.ErrorFieldMessage
+    ),
+  {
+    ssr: false,
+  }
+)
 
```

Comment:
Testar usando type:
```
export type InputProps = React.InputHTMLAttributes<HTMLInputElement> & {
  errorMessage?: string
  inputRef?: (node: HTMLInputElement | null) => void
}
```

Reply:
Fixed in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/43/commits/17b6cad8efe25edb11aa873c9aeb288047ff229b)